web/src/compose_state.js

```diff
@@ -1,6 +1,7 @@
 import $ from "jquery";
 
 import * as compose_pm_pill from "./compose_pm_pill";
+import * as compose_ui from "./compose_ui";
```

Comment:
As @andersk noted in a previous review, it is an import cycle to import `compose_ui` from here, so we shouldn't do that.

I'm not sure what the best fix is; one idea would be to have `export let compose_stream_widget` be defined here (but the value populated by a setter inside `compose_ui.js`); another would be to have `on_compose_select_stream_update`, which I think is called whenever that input's value is changed, be responsible for calling `set_stream_name` to update this data structure, and the couple of places that call `set_stream_name` be changed to call a `compose_ui.compose_stream_widget.something` method to actually modify the UI state and thus trigger this data state change?